### PR TITLE
feat: add arg `CROSS_ARCH` for cross-compilation

### PR DESCRIPTION
**Pull request by homalozoa**

### CODE REVIEW

**Comment by JEnoch** [comment]
Thanks for this contribution @homalozoa !
I didn't know about this [cargo-zigbuild](https://github.com/rust-cross/cargo-zigbuild) project. Seems pretty cool!

However, trying your instructions with ROS 2 Humble on Ubuntu 22.04 I get the following issues:
 1. `sudo apt install zig` returns `Unable to locate package zig`
     Maybe it's better to point to the official [zig install instructions](https://ziglang.org/learn/getting-started/#installing-zig)
 2. The `zigbuild` shall be also pre-installed.
 3. The Rust target shall also be pre-installed.

Could you please update your instructions in README.md accordingly, thus a "non-Rust-aware" ROS user won't encounter those issues?

File: README.md [hunk 1]
```diff
@@ -146,6 +146,18 @@ colcon build --packages-select zenoh_bridge_dds --cmake-args -DCMAKE_BUILD_TYPE=
...
 The `rosdep` command will automatically install *Rust* and *clang* as build dependencies.
 
+If you want to cross-compile the package on x86 device for any target, you can use the following command:
+```bash
+rosdep install --from-paths . --ignore-src -r -y
+colcon build --packages-select zenoh_bridge_dds --cmake-args -DCMAKE_BUILD_TYPE=Release  --cmake-args -DCROSS_ARCH=<target>
+```
+where `<target>` is the target architecture (e.g. `aarch64-unknown-linux-gnu`). The architechture list can be found [here](https://doc.rust-lang.org/nightly/rustc/platform-support.html).
+
+The cross-compilation uses `zig` as a linker. You can install it with:
```
Comment (JEnoch) [inline README.md:156]: Please, update instructions according to [discussion](https://github.com/eclipse-zenoh/zenoh-plugin-dds/pull/147#issuecomment-1697036587)

**Review by JEnoch — CHANGES_REQUESTED** [review]
Please, update requirements instructions

**Comment by homalozoa** [comment]
> Could you please update your instructions in README.md accordingly, thus a "non-Rust-aware" ROS user won't encounter those issues?

Sure! I've check the installation and update the README.

File: README.md (hunk 1 above)
Comment (homalozoa) [reply README.md]: Done. @JEnoch

**Review by JEnoch — APPROVED** [review]

**Pull request merged**